DOCS-3629: Add Metadata methods (#4159)
viamrobotics/docs: docs/dev/reference/changelog.md

<!-- If there is no concrete date for a change that makes sense, use the end of the month it was released in. -->

{{% changelog color="added" title="Store metadata" date="2025-03-28" %}}

> You can store and retrieve arbitrary data for your organization, location, machine, and machine part with the [Fleet Management API](/dev/dev/reference/apis/fleet/).

Author: ```suggestion
You can store and retrieve arbitrary metadata about your organization, location, machine, and machine part with the [Fleet Management API](/dev/dev/reference/apis/fleet/).
```

> You can store and retrieve arbitrary metadata about your organization, location, machine, and machine part with the [Fleet Management API](/dev/dev/reference/apis/fleet/).

Author: ```suggestion
You can store and retrieve arbitrary metadata about your organization, location, machine, and machine part with the [Fleet Management API](/dev/reference/apis/fleet/).
```
